.github/workflows/tests.yml

```diff
@@ -8,7 +8,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        ruby: [2.6, 2.7, 3.0, 3.1]
+        ruby: [2.7, 3.0, 3.1]
```

Comment:
Sorry -- didn't catch this earlier. we should also set `required_ruby_version` to `">= 2.7" in the gemspec.